packages/wrangler/src/__tests__/https-options.test.ts

```diff
@@ -1,6 +1,7 @@
 import fs from "node:fs";
 import os from "node:os";
-import { resolve } from "node:path";
+import path, { resolve } from "node:path";
+import { getConfigPath } from "../config-path";
```

Comment:
let's call this `getWranglerConfigPath`, and rename the file to `wrangler-config-path.ts`